scenes/quests/story_quests/stella/0_stella_intro/stella_intro.tscn

```diff
@@ -2,7 +2,7 @@
 
 [ext_resource type="TileSet" uid="uid://07fq3rspk8ia" path="res://scenes/tileset.tres" id="1_k0kvg"]
 [ext_resource type="Texture2D" uid="uid://dtm6xbnvtibk5" path="res://scenes/quests/story_quests/stella/0_stella_intro/stella_intro_components/stella_orion.png" id="2_vak0f"]
-[ext_resource type="SpriteFrames" uid="uid://dn0ivusgamrg3" path="res://scenes/quests/story_quests/stella/stella_player_components/stella_player.tres" id="3_vak0f"]
+[ext_resource type="SpriteFrames" uid="uid://t34p1kiw610y" path="res://scenes/quests/story_quests/pachamama/pachamama_player_components/pachamama_player.tres" id="3_vak0f"]
```

Comment:
Este cambio probablemente no fue intencional. Está cambiando una story quest que no es en la que estás trabajando: pachamama.

This change is probably not intended. It is changing a story quest that is not the one you are working on: pachamama.